conftest.py

```diff
@@ -149,7 +149,7 @@ def make_valid_request(send_post):
     def _make_valid_request(method: SensorMethod, params: dict | None = None) -> dict:
         payload = make_valid_payload(method=method, params=params)
         sensor_response = send_post(**payload)
-        return sensor_response.get("result", {})
+        return sensor_response.get("result", {}) or sensor_response.get("error", {})

```

Comment:
The way this function is currently implemented, it would respond with a dict containing sensor info if it executes successfully and with an empty dict if sensor returns an error. So, we could check whether the sensor returned an error without making changes to this function.